katas/metal-slug/andres/weapon.test.js

```diff
@@ -0,0 +1,8 @@
+import { Weapon } from './weapon.js';
+
+test('create shotgun object successfully', () => {
+    const obj = new Weapon('Shotgun', 5, true);
```

Comment:
Try to avoid using magic numbers when passing arguments to functions, in this case `5` and `true` do not have a meaningful way to know what they are. You can use constants/variables for them